hoomd/hpmc/update.py

```diff
@@ -466,72 +467,54 @@ def N(self):  # noqa: N802 - allow N as a function name
         return N_dict
 
 
-class remove_drift:  # noqa - will be rewritten for v3
+class RemoveDrift(Updater):
     """Remove the center of mass drift from a system restrained on a lattice.
 
     Args:
-        mc (:py:mod:`hoomd.hpmc.integrate`): MC integrator.
-        external_lattice (:py:class:`hoomd.hpmc.field.lattice_field`): lattice
-          field where the lattice is defined.
-        period (int): the period to call the updater
-
-    The command hpmc.update.remove_drift sets up an updater that removes the
-    center of mass drift of a system every period timesteps,
+        reference_positions ((*N*, 3) `numpy.ndarray` of ``numpy.float64``): the
+            reference positions of the lattice :math:`[\\mathrm{length}]`.
+        trigger (`hoomd.trigger.Trigger`): Select the timesteps to remove drift.
 
-    Example::
+    During the time steps specified by *trigger*, particle positions are
+    modified such that the their center of mass coincides with that of
```

Comment:
Correct the phrasing here. The center of mass does not enter into this operation.

Additionally, following the guidance of #1078, please add LaTeX math that describes the computation of the drift. Use the same notation as #1079 for consistency (here, `\vec{r}_i` for particle positions and `\mathrm{N_particles}` for the number of particles). I haven't introduced notation for the box minimum image yet, one idea would be `\mathrm{min_image}(\vec{a})`.